frontend/src/test/scala/bloop/tasks/TestUtil.scala

```diff
@@ -228,7 +228,8 @@ object TestUtil {
       testFrameworks = Array.empty,
       testOptions = Config.TestOptions.empty,
       javaEnv = javaEnv,
-      out = AbsolutePath(baseDirectory) // This means nothing in tests
+      out = AbsolutePath(baseDirectory), // This means nothing in tests
+      analysisOut = AbsolutePath(baseDirectory.resolve(s"$name-analysis.bin"))
```

Comment:
Can we have something like the now removed `ResultsCache.pathToAnalysis` but move it to the `jsonConfig` module and reuse it both here and in all the integration code?

Reply:
I didn't add dependency of config to `bloop.io` for `AbsolutePath` so I just created a method to create analysis file name there. Let me know what you think.